.gitlab/ci/.gitlab-ci.sdk-nightly.yml

```diff
@@ -256,31 +319,181 @@ demisto-sdk-nightly:run-end-to-end-tests:
     - name: docker.art.code.pan.run/build-tools--image-dind:20.10.12-dind
       alias: docker
   variables:
-    DOCKER_HOST: tcp://docker:2375
-    DOCKER_DRIVER: overlay2
-    DOCKER_TLS_CERTDIR: ""
-    PRODUCT_TYPE: "XSOAR"
-    SERVER_TYPE: "XSOAR"
-    INSTANCE_ROLE: "Server Master"
+      CLOUD_MACHINES_TYPE: "build"
+      CLOUD_MACHINES_COUNT: 1
+      INSTANCE_ROLE: "XSIAM"
+      GCS_QUEUE_FILE: "queue-ga"
+      GCS_LOCKS_PATH: "content-locks/locks-xsiam-ga"
+      CLOUD_SERVERS_FILE: "xsiam_servers_path"
+      XSIAM_SERVERS_PATH: "./xsiam_servers.json"
+      CLOUD_API_KEYS: $XSIAM_API_KEYS
+      CLOUD_API_TOKENS: $XSIAM_TOKENS
+      ARTIFACTS_FOLDER: "${ARTIFACTS_FOLDER_MPV2}"
+      ARTIFACTS_FOLDER_INSTANCE: "${ARTIFACTS_FOLDER_MPV2}/instance_xsiam"
+      GCS_MARKET_BUCKET: "${GCS_MARKET_V2_BUCKET}"
+      ENV_RESULTS_PATH: "${ARTIFACTS_FOLDER_SERVER_TYPE}/env_results.json"
```

Comment:
move it to be defined after the definition of ARTIFACTS_FOLDER_SERVER_TYPE on line 344